include/actuator/ActuatorParsing.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#ifndef ACTUATOR_PARSING_H_
+#define ACTUATOR_PARSING_H_
+
+#include <NaluParsing.h>
```

Comment:
Remove header and forward declare `YAML::Node`

```C++
namespace YAML { class Node; }
```

Reply:
ack